src/editor/body/EditorBody.js

```diff
@@ -88,8 +90,20 @@ class EditorBody extends Component {
 
         //Store the current markdown value in the browser local storage
         localStorage.setItem('markdownValue', value);
+        this.props.store.characterCount = value.length;
```

Comment:
da du 3 mal `this.props.store` schreibst, lohnt es sich daraus eine Variable zu machen.
im Constructor: `this.mobxStore = this.props.store`
oben in `componentDidMount`: `{ characterCount, wordsCount  } = this.mobxStore`
hier dann: `characterCount = value.length`

Reply:
Dadurch würde der Footer nicht mehr auf die Werte im Store zugreifen können, da die EditorBody Komponente eine eigene Kopie des Stores als Member Variable hätte.